auto_rx/autorx/templates/index.html

```diff
@@ -338,6 +396,12 @@
             }, sonde_follow_timeout);            
 
 
+            // Update telemetry table every 500ms
```

Comment:
This comment seems wrong. Seems to be every second.

Do we do this because we don't want the table updating so often that the user can't read the data in it if it's constantly changing?